linalg/blockmatrix.cpp

```diff
@@ -320,6 +321,46 @@ void BlockMatrix::EliminateRowCol(Array<int> & ess_bc_dofs, Vector & sol,
    }
 }
 
+void BlockMatrix::EliminateRowCols(const Array<int> & vdofs, BlockMatrix *Ae,
+                                   DiagonalPolicy dpolicy)
```

Comment:
The variable `policy` is not used in this function.  Is this intentional?

Reply:
@cjvogl, nice catch. I just realized that I introduced this bug while I was cleaning up this PR. I'll push the fix shortly.

Reply:
fixed